### PR TITLE
Build instructions

### PR DESCRIPTION
**Pull request by weliveindetail**
Readme: build instructions für homebrew und gtest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,19 @@ protobuf message as fast as possible.
...
 
+Debian:
 ```
 apt-get install build-essential cmake protobuf-compiler libprotobuf-dev libprotoc-dev libyajl-dev
 ```
 
-Then build:
+Mac OS X with Homebrew:
+```
+brew install gcc cmake protobuf protobuf-c yajl
```
Comment (0x7f) [inline README.md:23]: Are you sure `protobuf-c` is required? This is the protobuf compiler that generates C code which is not used in this project.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,17 @@ Mac OS X with Homebrew:
 brew install gcc cmake protobuf protobuf-c yajl
 ```
 
+If you don't have `gtest` installed, do it now:
```
Comment (0x7f) [inline README.md:26]: What kind of compiler/linkeder error did you experience? It should be cloned automatically into the build folder: https://github.com/0x7f/protog/blob/48f97c6d98b12a3a3e17aa2c9325f6de98aa0df0/test/CMakeLists.txt#L8-L19

File: README.md (hunk 1 above)
Comment (weliveindetail) [inline README.md:23]: No not sure, just a guess from your `protobuf-compiler` arg to `apt-get`

File: README.md (hunk 2 above)
Comment (weliveindetail) [inline README.md:26]: Ok right I see. Somehow cmake didn't do that.. `test/src/googletest` exists but it is empty. Tried it again and get:

```
$ make
Scanning dependencies of target protog
[  5%] Building CXX object CMakeFiles/protog.dir/src/protog.cpp.o
[ 10%] Linking CXX executable protog
[ 10%] Built target protog
[ 15%] Generating nestedmessage_parser.pb.cc, nestedmessage_parser.pb.h
[ 21%] Running C++ protocol buffer compiler on messages.proto
[ 26%] Generating simplemessage_parser.pb.cc, simplemessage_parser.pb.h
Scanning dependencies of target protog_test
[ 31%] Building CXX object test/CMakeFiles/protog_test.dir/test_nested_message.cpp.o
[ 36%] Building CXX object test/CMakeFiles/protog_test.dir/test_simple_message.cpp.o
[ 42%] Building CXX object test/CMakeFiles/protog_test.dir/messages.pb.cc.o
[ 47%] Building CXX object test/CMakeFiles/protog_test.dir/simplemessage_parser.pb.cc.o
[ 52%] Building CXX object test/CMakeFiles/protog_test.dir/nestedmessage_parser.pb.cc.o
make[2]: *** No rule to make target `test/src/googletest-build/googlemock/gtest/libgtest.a', needed by `test/protog_test'.  Stop.
make[1]: *** [test/CMakeFiles/protog_test.dir/all] Error 2
make: *** [all] Error 2
$ cmake --version
cmake version 3.5.2
```

**Pull request closed without merging**